core/src/components/datetime/datetime.scss

```diff
@@ -357,6 +357,11 @@ ion-picker-column-internal {
 }
 
 :host .calendar-day:after {
+  content: " ";
+}
+
+:host .calendar-day:after,
```

Comment:
~ Nit: Can we remove the `:host` style selectors from these additions? We have an open PR/task to remove it from the existing implementation (https://github.com/ionic-team/ionic-framework/pull/26383), but there are some odd difs that need to be worked through in that PR. 

`:host` shouldn't be necessary since we are encapsulated within the shadow dom here.